src/components/disappearing-header/index.tsx

```diff
@@ -44,13 +41,18 @@ type Props = {
   isRefreshing?: boolean;
   isFullHeight?: boolean;
 
+  leftButton?: LeftButtonProps;
+
+  tabPressBehaviour?: {
```

Comment:
Denne er gruppert inn for mer korrekt typedefinisjon uten for avanserte union types. Konsekvens er at man må være litt mer nøye i hook dependency checks som er lengre ned her.